src/Microsoft.OData.Edm/Microsoft.OData.Edm.csproj

```diff
@@ -95,6 +95,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.Data.OData" Version="5.8.4" />

```

Comment:
>Include="Microsoft.Data.OData" Version="5.8.4" /> [](http://example.com/codeflow?start=22&length=49)

Why do we add dependency for OData v3 lib? It doesn't make sense.

Reply:
I'm not sure how this line got added. Should I just remove it?